ui2/src/common/TableTransfer.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { Transfer } from 'antd';
+import _ from 'lodash';
+import { AppTable } from './Table';
+import { Link } from 'react-router-dom';
+import { AppButton } from '../common/Button';
+import { tableColumns } from '../utils/tableColumns';
+import './TableTransfer.css';
+
+export const TableTransfer = ({
+  data,
+  searchColumns,
+  rightColTitles,
+  leftColTitles,
+  selectGroup,
+  predefinedGroups,
+  linkTo,
+  addSearch,
+  ...props
+}) => {
+  const [targetKeys, setTargetKeys] = useState([]);
+  const [showSearch] = useState(!!searchColumns);
+  const formattedData = data.map(dataItem => ({ ...dataItem, key: dataItem.id.toString() }));
```

Comment:
why do you need this?

Reply:
data.map(({id, ...rest}) => ({ ...resy, key: id.toString() })); 😀

Reply:
I need this because antd demand key property of type string.
Fixed it!